_layouts/home.html

```diff
@@ -152,6 +152,28 @@ <h1 class="welcome-header orange-bg">
   ga('create', 'UA-92787920-1', 'auto');
   ga('send', 'pageview');
 
+</script><script>
+
+
+
+var countClicks = (function(){
```

Comment:
Is it just me, or is countClicks never used? You need to set an `onclick` handler somewhere.